apps/src/applab/api.js

```diff
@@ -328,8 +328,13 @@ export function startWebRequest(url, func) {
   return Applab.executeCmd(null, 'startWebRequest', {url: url, func: func});
 }
 
-export function startWebRequestSync(url, headers, func) {
-  return Applab.executeCmd(null, 'startWebRequestSync', {url, func, headers});
+export function startWebRequestSync(url, func) {
+  return Applab.executeCmd(null, 'startWebRequest', {
+    url: url,
+    func: (status, contentType, responseText) => {
+      func(responseText);
```

Comment:
just making sure i understand -- this means that in a user's project, the code would look like...

```javascript
var myResponseText = startWebRequestSync(myUrl);
```

...right?

Reply:
Yeah, that's correct